### DISCUSSION TITLE
Support for Minestom's Extensions

### DISCUSSION THREAD

**Discussion by lisbonjoker** [modrinth/.github#128] [Feature requests]
In the spirit of Open Source, a place for people to share and promote [Minestorm's](https://minestom.net/) [extensions](https://minestom.net/extensions/).

**Comment by triphora** [comment]
"Support for other games" is on the roadmap. We prefer not to have issues/discussions open for roadmap items.

**Reply by lisbonjoker** [reply]
> "Support for other games" is on the roadmap. We prefer not to have issues/discussions open for roadmap items.

It's not technically *another game*, It's still Minecraft but a Minecraft server implementation, open-source and without any code from Mojang. So it does fall in the current supported game.

**Reply by Hepno** [reply]
Minestom is not another game, it's a Minecraft Server Implementation that's Multithreaded & uses none of the vanilla NMS code.
